group_doc

```diff
@@ -0,0 +1,35 @@
+#-----------------------------------------------------Part 2; Exercise #1
+#submitted by: Laura
+#updated by:
+#verified by: *needs review*
+
+SELECT product_code, product_name, list_price, discount_percent
+  FROM products;
+  
+SELECT product_code, product_name, list_price, discount_percent
+  FROM products
+ ORDER BY list_price;  
+ 
+#------------------------------------------------------------------------
+
+#-----------------------------------------------------Part 2; Exercise #2
+#submitted by: Laura
+#updated by:
+#verified by: *needs review*
+
+SELECT CONCAT(last_name, ', ', first_name) AS full_name
+  FROM customers
+ WHERE LEFT(last_name, 1) BETWEEN 'M' and 'Z'
+ ORDER BY last_name;
+ 
+#------------------------------------------------------------------------
+
+#-----------------------------------------------------Part 2; Exercise #3
+#submitted by: Laura
+#updated by:
+#verified by: *needs review*
+
+SELECT product_name, list_price, date_added
+  FROM products
+ WHERE list_price BETWEEN 500 AND 2000
+ ORDER BY date_added DESC;
```

Comment:
"WHERE list_price BETWEEN 500 AND 2000" includes 500 and 2000. While this still returns the correct values for this particular query/table I believe "WHERE list_price > 500 AND list_price < 2000" would be more accurate since the question specifically asks for values greater than 500 and less than 2000. Thoughts?

Reply:
I reviewed 'BETWEEN' in mySQL and I agree with the change. thank you.